soca/obs/sss_trak_fnmoc.yaml

```diff
@@ -30,6 +30,13 @@
     where:
     - variable: {name: sea_surface_temperature@GeoVaLs}
       minvalue: 10.0
+  # Passivate obs where ocean fraction is > 90%
```

Comment:
... Oops ... Looks like I was not paying attention, these should have been passive for sure. We need to be a bit more careful when we put these yaml files together. Ouch.